etna/articles/models.py

```diff
@@ -114,7 +125,9 @@ class ArticlePage(HeroImageMixin, ContentWarningMixin, BasePageWithIntro):
         index.SearchField("article_tag_names"),
     ]
 
-    new_label_end_date = datetime.now() - timedelta(days=21)
+    @cached_property
+    def new_label_end_date(self):
+        return datetime.now().date() - timedelta(days=21)
```

Comment:
On the ticket, I suggested making the number of days an attribute on the mixin class, so that it can be overridden for individual types where needed. What do you think about that?